package.json

```diff
@@ -11,9 +11,21 @@
     "url": "https://github.com/storybookjs/addon-svelte-csf"
   },
   "license": "MIT",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "browser": "dist/cjs/index.js",
```

Comment:
You say this works in storybook?
I would expect this to be broken, but honestly I've not yet tried, so maybe I'm wrong.

What i know of the preset-loading mechanism in storybook is what when an addon-entry is added, storybook will try resolving `${name}/preset` (with node).
It will do the same for `manager` and `preview` FYI.

Maybe with `type: module` these exports fields are actually respected/used?

Reply:
Yeah, `type: "module"` is the key there as far as I understand it, which is mostly from [sindresorhus's explanation](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#faq)...

But yes, working in Node 16+ with Storybook 7.1+ in the repo.